### ISSUE TITLE
ch2 recipe2 rps_classifier error

### ISSUE THREAD

**Issue by zhaoyun0071** [PacktPublishing/Tensorflow-2.0-Computer-Vision-Cookbook#1]
ch2 recipe2 rps_classifier error

when set **loss=CategoricalCrossentropy(from_logits=True)**

you model should not have Softmax

**from_logits=True can auto calculate Softmax at the end of model**
